examples/jenkins/sample-app-masterslave-ci.json

```diff
@@ -0,0 +1,205 @@
+{
+  "kind": "Template",
+  "apiVersion": "v1",
+  "metadata": {
+    "name": "sample-app",
+    "annotations": {
+      "description": "This example shows how to create a simple ruby application in openshift origin v3",
```

Comment:
@bparees @mfojtik with the verdict on the master/slave stuff and removal of it in existing places already, I'm just going to remove these 2 json files I added as part of this pull.